lib/super_good/solidus_taxjar/backfill_transactions.rb

```diff
@@ -2,6 +2,13 @@ module SuperGood
   module SolidusTaxjar
     class BackfillTransactions
       def self.call
+        backfilled_orders = []
+        ::Spree::Order.complete.find_each do |order|
+          next if order.taxjar_order_transactions.any?
+          SuperGood::SolidusTaxjar.api.create_transaction_for(order)
+          backfilled_orders.push(order.number)
+        end
+        backfilled_orders
```

Comment:
If we found a way to query for the orders we could pass in the orders that need to be backfilled and we wouldn't need to have this return anything. We could loosen our coupling a bit.

Reply:
It would loosen the coupling because currently this receives an order from the controller and then returns the orders. Passing the orders would be looser because it wouldn't return anything so the coupling is only one direction. This isn't a blocker or anything but removing this return would be more of a Tell don't Ask implementation.